README.md

```diff
@@ -749,6 +749,7 @@ Table of Contents
   * [resizeappicon.com](https://resizeappicon.com/) — A simple service to resize and manage your app icons.
   * [smartmockups.com](https://smartmockups.com/) — Create product mockups, 200 free mockups.
   * [unDraw](https://undraw.co/) - A constantly updated collection of beautiful svg images that you can use completely free and without attribution.
+  * [userway.org](https://userway.org/) - Free Web Accessibility Widget. 
```

Comment:
please add something like `Free tier includes UserWay logo as well as all of the footer links, the donate link and more`

Reply:
what do you mean?